### ISSUE TITLE
[BUG] gcc not correctly supported in version 1.11.0

### ISSUE THREAD

**Issue by lustroid** [twibiral/obsidian-execute-code#340]
# Describe the bug
The most latest version 1.11.0 included support of gcc compiler for c (Great!!).  However it's not quite working. 
In the updated options UI, we are expected to see gcc being added to the Settings, according to changes in the source code. While these changes did not show up.
```
- .setName('Cling path')
- .setDesc('The path to your Cling installation.')
+ .setName('gcc / Cling path')
+ .setDesc('The path to your gcc / Cling installation.')
```
My current c++ settings page for 1.11.0:
![image](https://github.com/twibiral/obsidian-execute-code/assets/50304010/dc34d51b-5b82-4e05-875a-018904533ed6)

I also tried adding the correct gcc path to the Cling path box but it didn't seem to work.

# Software Version
Windows 11
Plugin: 1.11.0
Obsidian: 1.5.12

# To Reproduce
Update the plugin version to 1.11.0 and examine the c/c++ option UI.

# Expected behavior
gcc compiler being correctly supported.

![image](https://github.com/twibiral/obsidian-execute-code/assets/50304010/999b7005-2f86-4022-8e8e-942ba69b0aac)

**Comment by twibiral** [comment]
@melo-afk can you reproduce the issue?

**Comment by melo-afk** [comment]
Hi @twibiral , it seems like something with the build of the plugin went wrong. 

Screenshot of freshly installed the plugin via the community store
![release_build](https://github.com/twibiral/obsidian-execute-code/assets/58705527/6bc411cb-a3ae-4023-b670-ce4a4da86c60)

Screenshot of the plugin I built with `npm run dev` (master branch)
![manual_build](https://github.com/twibiral/obsidian-execute-code/assets/58705527/87b8cb4d-13b6-4ad8-b893-e8eb0f7c4b07)


@lustroid I only implemented the gcc functionality for C and not C++ as I didn't used C++ much. I can check in the coming weeks if I can get gcc or g++ with C++ to work

**Comment by twibiral** [comment]
Fixed with [version  1.11.1](https://github.com/twibiral/obsidian-execute-code/releases/tag/1.11.1). Thanks for reporting the bug! 

(It was indeed a problem with the build; I compiled the wrong version)
